helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/AvroCompatibilityHelper.java

```diff
@@ -76,6 +78,9 @@
           case AVRO_1_10:
             ADAPTER = new Avro110Adapter();
             break;
+          case AVRO_1_11:
+            ADAPTER = new Avro110Adapter(); //so far 1.11 is close enough to 1.10 for us to get away with this
```

Comment:
Let's not do this. It's going to confuse the hell out of people, e.g., when they see stack traces with source filename info. Maybe it will take some more time to do a proper copy-and-replace, but it'll be worth the wait. Nobody is crying out for 1.11 support in the compat helper just yet.